firmware/spike essential - v1.0.00.0070-51a2ff4 - 3.1.21/modules.txt

```diff
@@ -0,0 +1,10 @@
+>>> help('modules')
+__main__          math              uhashlib          uselect
+_onewire          micropython       uheapq            ustruct
+_uasyncio         uarray            uio               usys
+builtins          ubinascii         ujson             utime
+cmath             ubluetooth        umachine          utimeq
+firmware          ucollections      uos               uzlib
+gc                uctypes           urandom
+hub               uerrno            ure
+Plus any modules on the filesystem
```

Comment:
I've not bothered including the `Plus any modules on the filesystem` line in ther other files.